### DISCUSSION TITLE
ctrl 0 ( zero) moves cockpit view

### DISCUSSION THREAD

**Discussion by ppanther69** [flybywiresim/aircraft#6214] [General]
Hi,

after sim update 7, in VR, can´t use mouse in toolbar and windows, by pressing ctrl zero it works fine, but if I do this in FBW cockpit, the view moves to an inoperable position down in the captains seat.
Pressing ctrl zero again, the view moves to normal position, but the mouse doesn´t work again.
What I am doing wrong?

**Comment by wined2310** [comment]
you're not doing anything wrong. CTRL-0 is simply triggering the EFB View as it is the last one in the camera.cfg
With that VR Bug introduced with SU7, I hope we can change that to the default position, so the FBW can be used in VR "out-of-the-box".
 For now I manually edited the camera.cfg by overwrting the camera position of the EFB View with the default pilot view settings and it does the job.
But I'm afraid I will have to do this every time the DEV-build is getting updated and overwrites the camera.cfg again.
So would be nice, if this can be considered for future releases...
